### DISCUSSION TITLE
WG Core 2024-09-24

### DISCUSSION THREAD

**Discussion by emiltin** [rsmp-nordic/community#92] [Working Groups]
## GitHub Project
We started a new Project on core 3.3 at https://github.com/orgs/rsmp-nordic/projects/13/views/1, and will try to use this to organize work, and make it easier for members of the working group to help out on specific tasks.

**Comment by RazeAntoine** [comment]
@emiltin For the next session, I can work on this issue https://github.com/rsmp-nordic/rsmp_core/issues/128, thinking about the different possible solutions and their impact.

**Reply by emiltin** [reply]
Great, thank you! I'll assign you.
